cookbooks/aws-parallelcluster-config/resources/manage_fsx.rb

```diff
@@ -78,19 +91,43 @@
         dump 0
         pass 0
         options 'nfsvers=4.2'
-        action %i(mount enable)
```

Comment:
We are replicating the mount+enable steps for every file system type. What about avoiding this code duplication by putting the mount+enable logic within a function?

Reply:
They are not completely duplicate, Different filesystems are mounted with different mounting options. The code was like that before we support DFSM. Can we handle it outside this PR?

Reply:
Sure, let' address this in another Pr. I'll keep track of this